src/scripts/emailBreachAlerts.js

```diff
@@ -162,33 +163,38 @@ export async function poll(subClient, receivedMessages) {
           : [];
         /* c8 ignore stop */
 
-        const availableLanguages = process.env.SUPPORTED_LOCALES;
+        const availableLanguages = process.env.SUPPORTED_LOCALES.split(",");
```

Comment:
Oh, TIL that we have this env var. That's bound to get outdated at some point :sweat_smile: FWIW, on the website, [we check](https://github.com/mozilla/blurts-server/blob/1faee2378f0b9bcc514d08768dd81a18b08fe333/src/app/functions/server/l10n.ts#L29-L33) the subdirectories of `/locales/`, so we should be able to do something similar with `readdir` here. Out of scope for this PR I suppose, but wanted to get that out of my brain :)

Reply:
Yes, this is all based on the old Express app code... we really need to refactor it, the email templates are all written in a way to support both, we could surely simplify this.

Reply:
Ah, so `require.context()` isn't really a _Node_ thing, it's a _Webpack_ thing? https://webpack.js.org/guides/dependency-management/#requirecontext

Also, not sure what checking the subdirectories of `/locales/` would do, that'd just tell you available locales, not necessarily locales with 80%+ translations (which I think was the intent of `SUPPORTED_LOCALES`). Previously supported locales with high enough translation %%s were maintained in a separate SRE repo which was a bit awkward to maintain and required somebody to list added/removed locales in deploy tickets.

Reply:
Yep! And oh, that's a good point! I'd still like for that to be automated some way, but in lieu of that... That means that we are now shipping locales with <80% coverage as well :/

Reply:
Re: automation,
As with most things in life, I'd say @flodolo is much better equipped to answer that than I am.
But for automation, I built https://github.com/pdehaan/pontoonql a lifetime ago to query Pontoon's GraphQL endpoints for translation stats for a project.

```sh
npx pdehaan/pontoonql firefox-monitor-website 80

[{"locale":{"code":"zh-CN","name":"Chinese (China)"},"totalStrings":783,"approvedStrings":783,"stringsWithWarnings":0,"missingStrings":0,"progress":100}, ...]
```

That might be too verbose to be useful, so I tend to pipe that through `jq` to extract just the `locale.code`:

```sh
npx pdehaan/pontoonql firefox-monitor-website 80 | jq '[.[] | .locale.code]'
[
  "zh-CN",
  "zh-TW",
  "cs",
  "da",
  "nl",
  "en-CA",
  "fr",
  "de",
  "it",
  "pt-BR",
  "ru",
  "sk",
  "sl",
  "es-AR",
  "es-CL",
  "es-ES",
  "sv-SE",
  "uk",
  "cy",
  "fy-NL",
  "hu",
  "el",
  "ia",
  "vi",
  "en-GB",
  "fi",
  "gn",
  "tr",
  "id",
  "sq",
  "es-MX",
  "nn-NO",
  "pt-PT"
]
```

Although probably caveats caveats everywhere since I'm pulling from Pontoon and its probably missing reference locales, etc.

Not sure if there is a good way to add that into the pipeline to keep it up to date, or it might be suboptimal if a translation gets temporarily bumped down to 79.8% on a deploy day and gets removed from the `SUPPORTED_LOCALES` list until the next deploy in 2 weeks. Not sure if manual judgement is better (although equally fallible).